ui/src/album/albumLists.js

```diff
@@ -18,6 +19,12 @@ export default {
       params: 'sort=starred_at&order=DESC&filter={"starred":true}',
     },
   }),
+  ...(config.enableStarRating && {
+    rating: {
+      icon: StarIcon,
+      params: 'sort=rating&order=DESC',
```

Comment:
I just added a `has_rating` filter in the album_repository, you can set it with:
```
&filter={"has_rating":true)
```
So only rated albums will show up in this view